style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+.main {
+  position: relative;
+
+  height: 100vh;
+  width: 100vw;
+  background: black;
+}
+.main img {
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  width: 100%;
+  height: 100%;
+  
+  object-fit: cover;
+}
+.main img:nth-child(2) {
+  mix-blend-mode: multiply;

```

Comment:
can you explain this property ?

Reply:
Mix blend mode property shows how the element should mixed with the parent elements background color.